Front-End/challenge1/script.js

```diff
@@ -1 +1,16 @@
-// Modify this file only
+let counter = 0;
+
+const element_counter = document.getElementById("counter");
+
+decrease = () => {
+  counter != 0 ? (counter = counter - 1) : (counter = 0);
```

Comment:
Why it's necessary the assignation counter = 0 in line 6? If it was previously initialized on line 1?